model/labels/labels.go

```diff
@@ -452,7 +452,7 @@ func (b *Builder) Labels() Labels {
 
 	// In the general case, labels are removed, modified or moved
 	// rather than added.
-	res := make(Labels, 0, len(b.base))
+	res := make(Labels, 0, len(b.base)+len(b.add))
```

Comment:
Would have been good to update the comment which is explaining the previous behaviour.

Reply:
To be honest I think I've left this one out when cutting out PR and it's the other change I was going for.
I'm not sure how to validate claim from the comment or disprove it.